File: CodeWars/JavaScript/7Kyu/FindDuplicates.js
// URL: https://www.codewars.com/kata/find-duplicates/train/javascript

function duplicates(arr) {
  let duplicatesList= [];
  let numbersFound= [];

  for (i=0;i<arr.length;i++){
    if ((numbersFound.includes(arr[i])) && (!duplicatesList.includes(arr[i]))){
      duplicatesList.push(arr[i]);
      }
    else{
      numbersFound.push(arr[i]);
      }
    }
    
  return duplicatesList
}

console.log(duplicates([1, 2, 4, 4, 3, 3, 1, 5, 3, '5']));

File: CodeWars/Python/7Kyu/FindDuplicates.py
# URL: https://www.codewars.com/kata/find-duplicates/train/python

def duplicates(array):
  duplicatesList=[]
  numbersFound = []

  for index, item in enumerate(array):
    if (item in numbersFound) and (item not in duplicatesList):
      duplicatesList.append(item)
    else:
      numbersFound.append(item)
      
  return duplicatesList

print(duplicates([1, 2, 4, 4, 3, 3, 1, 5, 3, '5']))